std_msgs/msg/Empty.msg

```diff
@@ -0,0 +1,3 @@
+# Temporary dummy field to get around the fact that some IDL parsers (e.g.,
+# OpenSplice) don't like having empty messages
+uint8 dummy
```

Comment:
I don't know what you did when you ran into this but you don't have to add any fields to an .msg file to make it work with OpenSplice. This happens internally and is transparent to the user. Other ROS 2 packages do contain empty message definition files without a problem.